src/main/java/com/conveyal/gtfs/graphql/fetchers/SQLColumnFetcher.java

```diff
@@ -29,17 +39,55 @@
     public SQLColumnFetcher(String tableName, String parentJoinField, String columnName) {
         this.columnName = columnName;
         this.jdbcFetcher = new JDBCFetcher(tableName, parentJoinField);
-
     }
 
+    /**
+     * This get method does not use the JdbcFetcher get method because the query ultimately returns a list of values
```

Comment:
Perhaps this explains the changes elsewhere, where methods now return DataFetcher<Object> instead of a more specific type. I don't fully understand what has changed, or how the caller receiving results from these methods will know whether it is receiving a list of maps or a list of other objects, but it seems like the type could at least be DataFetcher<List<Object>>. Still feels like something is a bit amiss, but maybe it's with the GraphQL library rather than our code?